tools/CouchDB_MRI_Importer.php

```diff
@@ -88,14 +88,19 @@ function _generateCandidatesQuery($ScanTypes)
 
         foreach($s as $scan){
             $scantype=$scan['ScanType'];
-            $Query .= ", (SELECT f.File FROM files f LEFT JOIN mri_scan_type msc
-              ON (msc.ID= f.AcquisitionProtocolID)
-              WHERE f.SessionID=s.ID AND msc.Scan_type='$scantype' LIMIT 1)
-                    as Selected_$scantype, (SELECT fqc.QCStatus
-                    FROM files f 
+            $Query  .= ", (SELECT f.File FROM files f
+                    LEFT JOIN files_qcstatus fqc USING(FileID)
+                    LEFT JOIN mri_scan_type msc ON (msc.ID= f.AcquisitionProtocolID)
+              WHERE f.SessionID=s.ID AND msc.Scan_type='$scantype'
+              AND fqc.Selected='true' ORDER BY
+              FIND_IN_SET(fqc.QCStatus, 'Fail,Pass') DESC LIMIT 1)
+                    as Selected_$scantype,
+              (SELECT fqc.QCStatus FROM files f
                     LEFT JOIN files_qcstatus fqc USING(FileID)
                     LEFT JOIN mri_scan_type msc ON(msc.ID= f.AcquisitionProtocolID)
-              WHERE f.SessionID=s.ID AND msc.Scan_type='$scantype' LIMIT 1)
+              WHERE f.SessionID=s.ID AND msc.Scan_type='$scantype'
+              AND fqc.Selected='true' ORDER BY
+              FIND_IN_SET(fqc.QCStatus, 'Fail,Pass') DESC LIMIT 1)
```

Comment:
@gluneau FIND_IN_SET(fqc.QCStatus, 'Fail,Pass') will return `1` if fqc.QCStatus is either Fail or Pass. `0` otherwise. So it is not enforcing ordering by pass first